### DISCUSSION TITLE
CannotLoadViewerScript for PDF's

### DISCUSSION THREAD

**Discussion by aimanskie** [exoplatform/exo-community#63] [Q&A]
Hi, does anyone know where to find the api.js to link with exoplatform on load and have the preview of pdf embedded. Currently it is not showing. Appreciate any help on this, thanks!

<img width="429" alt="image" src="https://github.com/exoplatform/exo-community/assets/86446622/16a6306e-87ec-4891-b4f2-3e6f138313f6">

<img width="505" alt="image" src="https://github.com/exoplatform/exo-community/assets/86446622/12664282-5c7f-4e14-b1e1-d436e2d1a1fe">

**Comment by rdenarie** [comment]
cc @ahamdi 
As you work on this part for community version, can you answer to @aimanskie

**Comment by ahamdi** [comment]
@aimanskie onlyoffice addon is installed by default and need to be removed? this will be done in the next update of the exo community server
if you use the docker compose? you need just to add this property  https://github.com/exo-docker/exo-community/blob/master/docker-compose.yml#L25

**Reply by aimanskie** [reply]
@ahamdi i thought onlyoffice was for editing documents for the following,
Word (.docx)
Excel (.xlsx)
PowerPoint (.pptx)

However, for my case i just wanted to preview the PDF files. Are you saying, to have preview of pdf files i need Onlyoffice Server set up?

**Reply by aimanskie** [reply]
@ahamdi btw, can we set up the exo community together with the onlyoffice community? Exo community is already up, how do add the onlyoffice community and link with exo?

**Reply by ahamdi** [reply]
@aimanskie No need to have OO server to preview PDF files, that is why you will need just to uninstall it as I explained in the previous answered message

**Reply by aimanskie** [reply]
@ahamdi Thanks for the clarification. However, after adding the property the same preview error is showing (attached in main question).

**Reply by ahamdi** [reply]
@aimanskie could you try to recreate the container? the message is related to Only office and since it is still displayed it seems it is not removed from the container

**Reply by aimanskie** [reply]
We've tried but its still not working. I guess the OnlyOffice is still not removed. We're on version 6.4, maybe that's the issue? @ahamdi
BTW, is there any downside from upgrading to 6.5? Or any problems that may arise from us using the same databases with the new 6.5 version? It's off topic but if can try upgrading and also apply the docker-compose property to remove OnlyOffice, it may be worth the shot?

**Reply by aimanskie** [reply]
It's working with a fresh version of 6.4. and its working with version 6.5 :) Thank you! @ahamdi
